ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -610,9 +607,25 @@ class JITModBuilder {
     AllocaInst *ActiveFrameVec = FailBuilder.CreateAlloca(
         ActiveFrameSTy,
         ConstantInt::get(PointerSizedIntTy, ActiveFrames.size()));
+
+    std::vector<Value *> LiveValues;
     for (size_t I = 0; I < ActiveFrames.size(); I++) {
       FrameInfo FI = ActiveFrames[I];
 
+      // Read live AOT variables from stackmap calls and match them to
+      // variables in the JIT so we they can be optimised.
+      CallBase *SMC = FI.SMCall;
+      for (size_t Idx = 2; Idx < SMC->arg_size(); Idx++) {
+        Value *Arg = SMC->getArgOperand(Idx);
+        if (VMap.find(Arg) != VMap.end()) {
+          if (Arg == NewControlPointCall) {
```

Comment:
We need a comment saying why we skip the control point.

It's because we are only dealing with deopt sites, right?

Reply:
Fixed in [3a22adb](https://github.com/ykjit/yk/pull/562/commits/3a22adbce65409477aa8f4e83c0b5e17f6da2045)